pom.xml

```diff
@@ -214,23 +222,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
-                <version>${depends.plugin.version}</version>
-                <executions>
-                    <execution>
-                        <id>dependencies</id>
-                        <phase>prepare-package</phase>
-                        <goals>
-                            <goal>tree</goal>
-                        </goals>
-                        <configuration>
-                            <outputFile>mvn_dependency_tree.txt</outputFile>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
```

Comment:
If this has been removed, check it exists in the child poms

Reply:
Added this back into the parent pom